Source/bmalloc/bmalloc/CryptoRandom.cpp

```diff
@@ -75,7 +75,7 @@ class ARC4RandomNumberGenerator : public StaticPerProcess<ARC4RandomNumberGenera
     ARC4Stream m_stream;
     int m_count;
 };
-DECLARE_STATIC_PER_PROCESS_STORAGE(ARC4RandomNumberGenerator);
+DECLARE_STATIC_PER_PROCESS_STORAGE_WITH_LINKAGE(ARC4RandomNumberGenerator, BNOEXPORT);
```

Comment:
This looks scary - how do we enforce that it's per process if we don't export it?

If this is how we have to do it, that needs at least a comment explaining why it doesn't cause problems.

Reply:
Is this entirely correct? The "per process" naming implies to me that we don't want two of those to be brought in into the process. Which seems easier to do accidentally now.

Reply:
Maybe a better way to put this is: `ARC4RandomNumberGenerator` an implementation detail of `bmalloc::cryptoRandom()`. It's declared in this cpp file and is not exported. So it would not be possible to have a declaration of this class and its inlined storage-accessing methods outside of the compilation unit. This change gives the storage object the same visibility as the class itself.